static/src/components/builder/components/DropdownInput/dropdownTemplate.html

```diff
@@ -0,0 +1,5 @@
+<dropdown-input
+    pyb-answer="{{pybAnswer}}"
+    :choices="{{choices}}"
```

Comment:
If we set choices like this :choices='{{{choices}}}' then in the preview code the &quot will not be there and it will look like :choices='{"Op1":"Test op1"}'

Reply:
Maybe we can also change that on the NLP component for the :tags props

Reply:
I will work on this in next sprint.